### PR TITLE
Leathercraft desynth updates

### PR DESCRIPTION
**Pull request by FuzzyWereBeast**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: sql/synth_recipes.sql [hunk 1]
```diff
@@ -3113,7 +3113,7 @@ INSERT INTO `synth_recipes` VALUES (45009,0,0,0,0,0,50,103,0,0,0,4099,4241,869,2
 INSERT INTO `synth_recipes` VALUES (45010,0,0,0,0,0,0,104,0,0,0,4103,4245,635,3547,4509,0,0,0,0,0,3548,3548,3548,3548,1,1,1,1,'Sealord Leather');
 INSERT INTO `synth_recipes` VALUES (45011,0,0,0,0,0,0,104,0,0,0,4103,4245,695,3547,4509,0,0,0,0,0,3548,3548,3548,3548,1,1,1,1,'Sealord Leather');
 INSERT INTO `synth_recipes` VALUES (45012,0,0,0,0,0,0,104,0,0,0,4099,4241,1629,3552,3552,0,0,0,0,0,10546,10547,10547,10547,1,1,1,1,'Urja Gloves');
--- INSERT INTO `synth_recipes` VALUES (45013,0,0,0,255,255,0,104,0,0,0,4099,4241,665,862,862,914,1409,2189,2275,3925,27920,27921,27921,27921,1,1,1,1,'Pak Corselet');
+INSERT INTO `synth_recipes` VALUES (45013,0,0,0,62,255,0,104,0,0,0,4099,4241,665,862,862,914,1409,2189,2275,3925,27920,27921,27921,27921,1,1,1,1,'Pak Corselet');  -- 2021.11.17  FFXIclopedia and BGWiki show 104, JPWiki shows 104-105 Leathercraft.  BGWiki shows 60-62 Smithing.  All show ??? Goldsmithing.
```
Comment (Xaver-DaRed) [inline sql/synth_recipes.sql]: Leave this one commented out if it still has a 255 lvl

**Review by Xaver-DaRed — CHANGES_REQUESTED** [review]

File: sql/synth_recipes.sql (hunk 1 above)
Comment (TeoTwawki) [reply sql/synth_recipes.sql]: guessing typo and meant to edit that not the 0

Comment (FuzzyWereBeast) [reply sql/synth_recipes.sql]: No. It was an error on my end. There were a lot of 2 and 3 craft synths there. I didn't mean for that one to be uncommented. I'll get it fixed tonight.

Comment (FuzzyWereBeast) [reply sql/synth_recipes.sql]: Concerns corrected.

**Review by Xaver-DaRed — APPROVED** [review]

**Review by TeoTwawki — APPROVED** [review]

**Pull request merged**